workos/types/user_management/organization_membership.py

```diff
@@ -18,6 +19,6 @@ class OrganizationMembership(WorkOSModel):
     user_id: str
     organization_id: str
     role: OrganizationMembershipRole
-    status: OrganizationMembershipStatus
+    status: LiteralOrUntyped[OrganizationMembershipStatus]
```

Comment:
Nice catch.

I wonder if there are other literals like this? In FGA perhaps?

Reply:
Oh. Yup. Here: https://github.com/workos/workos-python/blob/f1dbfca2e9e9f2cb492fed7748e195dc65608bdd/workos/fga.py#L204

Reply:
I only see literals as inputs in FGA actually. So that's fine.

Reply:
Oh, but there is also this one in MFA: https://github.com/workos/workos-python/blob/f1dbfca2e9e9f2cb492fed7748e195dc65608bdd/workos/types/mfa/authentication_factor.py#L41

Reply:
Didn't see them at first, but found two others and fixed.